### ISSUE TITLE
Query

### ISSUE THREAD

**Issue by ali4321143** [SaadSubhaniTheProgrammerVersion/Textual-Report-Generation-from-Chest-X-rays-Indiana-Dataset#1]
Thanks for sharing the code
Kindly if you can answer these question
Q1 Does the initial reports you are generating by GRU? and then using the BIOGPT you are finetuning that generated report by GRU
Q2 The loss function for training the GRU is not there in the (create_model) file
Q2 How you are dealing with the variable length if you are feeding the pad token into the GRU then how you are ignoring its effect
Kindly if you can answer these queries
Thanks
